client/client-lib/src/mint/mod.rs

```diff
@@ -248,7 +251,7 @@ impl<'c> MintClient<'c> {
                 loop {
                     match self.fetch_coins(&mut dbtx, out_point).await {
                         Ok(_) => {
-                            dbtx.commit_tx().expect("DB Error");
+                            dbtx.commit_tx().await.expect("DB Error");
```

Comment:
There is an issue here causing the clientd integration test to fail because fetch is not returning the coins from the peg_in. Need to investigate more to determine the solution

Reply:
I fixed this by putting it inside of a futures::executor::block_on. Blocking inside of clientd was causing issues (threads were hanging and ecash could not be spent).